### PR TITLE
Switch to ci/fast for ec2 CI jobs

### PR DESCRIPTION
**Pull request by dims**
- the ci/ folder takes longer to populate ci/fast is quicker
- test-package-* are needed to be set to pick up e2e.test from the right spot.

### CODE REVIEW

**Review by xmudrii — APPROVED** [review]
/lgtm
/hold
if you want other folks to take a look before merging, otherwise unhold

**Comment by dims** [comment]
/hold cancel

thanks @xmudrii

**Pull request merged**
